### ISSUE TITLE
HOw to achieve 94.8% of Accuracy

### ISSUE THREAD

**Issue by wuchaowei2012** [tcapelle/action_recognition#10]
Dear Tcapelle. 

I clone this repo and find that I could only achieve 60% of Accuracy if I achieve ConvLSTM model. It is obvious that I overfit this model,

Could you share the training procedure.

Have  a  good day.

**Comment by tcapelle** [comment]
You get this with a random split 80/20, with the split0 (what is used on the notebook, you get 60%).

**Comment by wuchaowei2012** [comment]
Which one is 94%, top1 accuracy, or top-5 accuracy?

**Comment by tcapelle** [comment]
top1

**Comment by wuchaowei2012** [comment]
Bravo! By the way, how many epochs does it spend and the corresponding learning rate?

**Comment by tcapelle** [comment]
it is training right now, you will have to wait, I am squeezing some GPU between my models.

**Comment by wuchaowei2012** [comment]
Did you train an epoch of this model and then change the learning rate in the next epoch?

**Comment by wuchaowei2012** [comment]
Did you train an epoch of this model and then change the learning rate int the next epoch?

**Comment by tcapelle** [comment]
Look at my last commit, you have the training.
I am unable to reproduce the 94% on 10 epochs, will have to try later with more epochs.more layers.
The learning rate is scheduled with a One Cycle policy.

**Comment by tcapelle** [comment]
@wuchaowei2012  you moativated me, I am re-running everything...

**Comment by tcapelle** [comment]
I retrained and added the top5 accuracy metrics.
